squad/frontend/templates/squad/tests.jinja2

```diff
@@ -32,7 +32,11 @@
             <tr id='test-{{test.name|slugify}}' ng-show="match('test-{{test.name|slugify}}')">
                 <td><a href="{{url('test_history', args=[build.project.group.slug, build.project.slug, test.name])}}">{{test.name}}</a></td>
                 {% for status in test %}
-                <td class="{{status}}"><a href="{{url('test_history', args=[build.project.group.slug, build.project.slug, test.name])}}">{{status}}</a></td>
+                <td class="{{status[0]}}"><a href="{{url('test_history', args=[build.project.group.slug, build.project.slug, test.name])}}">{{status[0]}}</a>
+                  {% if status[1] %}
+                      <a href='#' class='pull-right' data-toggle="modal" data-target="#info_modal" data-status="{{ status[1] }}"><span class='fa fa-info-circle'></span> Show info</a>
```

Comment:
I would leave only the icon, and set  "Show info" as a tooltip. if you have many columns, several "show info"s will be a great distraction; see for example the amount of columns in https://qa-reports.linaro.org/lkft/linux-stable-rc-4.14-oe/build/v4.14.93/tests/